### DISCUSSION TITLE
Help: Bricked Retroarch on 280V - Controls not working anymore

### DISCUSSION THREAD

**Discussion by Santorayo3** [eduardofilo/RG350_adam_image#150] [RetroArch topics]
I changed inputs for the left analog stick to the d-pad to be able to move the mouse cursor in the Amiga emulator. (R280 doesn't have an analog stick). That worked but then I saved it as an override (not in the core settings but in general)

Now I can't navigate the RetroArch menu anymore... not even Select+Start to exit works anymore. When I delete the retroarch.cfg file a new one is greated and it says "Dinus Controller Set Port#1" or something similar but it still doesn't work. RetroArch also turned green... I think it used to be red before that.

Can I get a proper retroarch.cfg or repair it without re-installing the whole image?

**Comment by eduardofilo** [comment]
Yes, the red one config screen is the way to know that the configuration belongs to Adam. The green one is a fresh RetroArch config.
I attach a Py Backup file with the fresh config of RetroArch in Adam 1.3.1. Just copy it to backups folder in EXT card, and restore it with Py Backup selecting only the `RetroArch config` entry.

Remove the `.zip` extension. I had to put it because Github won't let me attach tgz files.

If after restoring you still have problems, delete the `config` directory inside `.retroarch` and try again.

[RetroArch config.tgz.zip](https://github.com/eduardofilo/RG350_adam_image/files/7810187/RetroArch.config.tgz.zip)

**Comment by Santorayo3** [comment]
Thanks. I did that. Now RetroArch menu is again shown in red but I still can't interact with it. None of the keys do anything. 😶

**Reply by eduardofilo** [reply]
Did you delete the config directory before loading the backup?

**Comment by Santorayo3** [comment]
Got it working again. I had to delete the autoconfig folder as well.

**Reply by eduardofilo** [reply]
Good to know. I was not aware of the effect of that directory. I've always seen it empty. I suppose it has to do with the setting in `Settings > Input > Autoconfig`, but I'll have to do some research. Thanks for finding the solution.
